File: endpoint/templates/node/express/babel/endpoint.route.js
import <%= name %>Controller from '../controller/<%= name %>-controller';

export default class <%= name %>Routes {
  static init(router) {
    router
      .route('/api/<%= nameLowerCase %>')
      .get(<%= name %>Controller.getAll)
      .post(<%= name %>Controller.createNew);

    router
      .route('/api/<%= nameLowerCase %>/:id')
      .delete(<%= name %>Controller.removeById);
  }
}

File: endpoint/templates/go/gin/endpoint.route.go
package <%= nameLowerCase %>routes

import (
	"<%= repoHostUrl %>/<%= userNameSpace %>/<%= appName %>/server/api/<%= feature %>/controller"
	"gopkg.in/gin-gonic/gin.v1"
)

func Init(r *gin.Engine) {
	r.GET("/api/<%= nameLowerCase %>", <%= nameLowerCase %>controller.GetAll)
	r.GET("/api/<%= nameLowerCase %>/:id", <%= nameLowerCase %>controller.GetById)
	r.POST("/api/<%= nameLowerCase %>", <%= nameLowerCase %>controller.New)
	r.PUT("/api/<%= nameLowerCase %>/:id", <%= nameLowerCase %>controller.Update)
	r.DELETE("/api/<%= nameLowerCase %>/:id", <%= nameLowerCase %>controller.Remove)
}
